### ISSUE TITLE
Update:  schedule session.interact_user_follow amount higher then 1 is broken?

### ISSUE THREAD

**Issue by GitNees** [InstaPy/InstaPy#2970]
UPDATE:

So I did more tests, it keeps running now perfectly since i changed session.interact_user_followers amount=1, anything above 1 makes it crash next session. Also when running amout=10 will only do 1 action. I dont understand it because ONLY running interact_user_followers with higher amount, then schedule doesn't crash.

------------------------

------------------------

Actually, I don't know what causes this at all. Used to work on older versions.

`ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

Traceback (most recent call last):
  File "C:\Python\InstaPy-master\quickstart.py", line 76, in <module>
    schedule.run_pending()
  File "C:\Python\lib\site-packages\schedule\__init__.py", line 493, in run_pend
ing
    default_scheduler.run_pending()
  File "C:\Python\lib\site-packages\schedule\__init__.py", line 78, in run_pendi
ng
    self._run_job(job)
  File "C:\Python\lib\site-packages\schedule\__init__.py", line 131, in _run_job

    ret = job.run()
  File "C:\Python\lib\site-packages\schedule\__init__.py", line 411, in run
    ret = self.job_func()
  File "C:\Python\InstaPy-master\quickstart.py", line 45, in job
    session.interact_user_followers(['randomusername'], amount=100, ran
domize=True)
  File "C:\Python\InstaPy-master\instapy\instapy.py", line 2181, in interact_use
r_followers
    self.logfolder)
  File "C:\Python\InstaPy-master\instapy\unfollow_util.py", line 881, in get_giv
en_user_followers
    channel, jumps, logger, logfolder)
  File "C:\Python\InstaPy-master\instapy\unfollow_util.py", line 695, in get_use
rs_through_dialog
    person_list = dialog_username_extractor(buttons)
  File "C:\Python\InstaPy-master\instapy\unfollow_util.py", line 718, in dialog_
username_extractor
    if person and hasattr(person, 'text') and person.text:
  File "C:\Python\lib\site-packages\selenium\webdriver\remote\webelement.py", li
ne 76, in text
    return self._execute(Command.GET_ELEMENT_TEXT)['value']
  File "C:\Python\lib\site-packages\selenium\webdriver\remote\webelement.py", li
ne 628, in _execute
    return self._parent.execute(command, params)
  File "C:\Python\lib\site-packages\selenium\webdriver\remote\webdriver.py", lin
e 312, in execute
    self.error_handler.check_response(response)
  File "C:\Python\lib\site-packages\selenium\webdriver\remote\errorhandler.py",
line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale elemen
t reference: element is not attached to the page document
  (Session info: chrome=69.0.3497.100)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12
b7),platform=Windows NT 6.3.9600 x86_64)

`

**Comment by GitNees** [comment]
Extra info:Line 76 =     schedule.run_pending()

from:

while True:
    schedule.run_pending()
    time.sleep(1)

**Comment by GitNees** [comment]
What I did see, was that the session ended later then the next run. In the past, this was never a problem, it would just start again. Now this happens.

**Comment by GitNees** [comment]
Still debugging. Tested that theorie. Changes all parmaters to a minumum so that it would finish quicker and set next schedule 1minute later. After 3mins, session ended, new session started again. So it is not that.

I seem to get the error related to session.interact_user_followers(['randomusername'], amount=100, ran
domize=True). I have now set it to 10 and still only does 1 simulated follow and stops, then the details show from the session and then the error appears . Now testing all normal paramterers except session.interact_user_followers amount set to amount 1. I have a feeling that isn't working as intended and does something odd.
